### PR TITLE
updates to the static layer HDF5

### PR DESCRIPTION
**Pull request by LiangJYu**
This PR updates the static layer HDF5 by:
* storing datasets under `data` not `data/static_layers`
* adding `identification` group
* removing `noise` and `calibration` in from `metadata` group

### CODE REVIEW

File: src/compass/s1_geocode_metadata.py [hunk 1]
```diff
@@ -115,9 +119,9 @@ def run(cfg, burst, fetch_from_scratch=False):
                 dtype = np.byte
 
             # Create dataset with x/y coords/spacing and projection
-            topo_ds = init_geocoded_dataset(static_layer_group, layer_name,
-                                            geo_grid, dtype,
-                                            np.string_(layer_name))
+            topo_ds = initdata_geocoded_dataset(static_layer_group, layer_name,
```
Comment (vbrancat) [inline src/compass/s1_geocode_metadata.py:122]:
```suggestion
            topo_ds = init_geocoded_dataset(static_layer_data_group, layer_name,
```
Also, for consistency, we should use `static_layers_data_group`

**Review by vbrancat — COMMENTED** [review]
- [x] 1. The `identification` group contains a `product_type` with a value CSLC-S1. This should be changed to `Static layers CSLC-S1`
- [ ] 2. For the data group we want to specify angle to all the datasets (`heading_angle,` `incidence_angle`, `local_incident_angle`). Refer to ISCE3 for correct nomenclature of these layers.
- [ ] 3. In `processing_information/algorithms` we should include the algorithm to generate the topo layers (radar to geo)
- [ ] 4. `processing_information/parameters` contains the flattening flags which should be removed. Any parameter we want to list? Otherwise we can remove it.
- [ ] 5. In the metadata, let's include the `inputs` too s that is clear all the info we used to generate the burst
- [ ] 6. I believe we should include also input_burst_metadata, in case someone wants to reconstruct the radar grid

The code fails at the generation of the QA:
```
Traceback (most recent call last):
  File "/mnt/aurora-r0/vbrancat/codes/COMPASS/src/compass/s1_cslc.py", line 54, in <module>
    main()
  File "/mnt/aurora-r0/vbrancat/codes/COMPASS/src/compass/s1_cslc.py", line 49, in main
    run(parser.run_config_path, parser.args.grid_type)
  File "/mnt/aurora-r0/vbrancat/codes/COMPASS/src/compass/s1_cslc.py", line 43, in run
    s1_geocode_slc.run(cfg)
  File "/mnt/aurora-r0/vbrancat/codes/miniconda3/envs/compass/lib/python3.11/site-packages/compass/s1_geocode_slc.py", line 123, in run
    s1_geocode_metadata.run(cfg, burst, fetch_from_scratch=True)
  File "/mnt/aurora-r0/vbrancat/codes/miniconda3/envs/compass/lib/python3.11/site-packages/compass/s1_geocode_metadata.py", line 152, in run
    cslc_qa.compute_static_layer_stats(h5_obj, cfg.rdr2geo_params)
  File "/mnt/aurora-r0/vbrancat/codes/miniconda3/envs/compass/lib/python3.11/site-packages/compass/s1_cslc_qa.py", line 134, in compute_static_layer_stats
    self.compute_stats_from_float_hdf5_dataset(cslc_h5py_root,
  File "/mnt/aurora-r0/vbrancat/codes/miniconda3/envs/compass/lib/python3.11/site-packages/compass/s1_cslc_qa.py", line 203, in compute_stats_from_float_hdf5_dataset
    qa_item_ds = cslc_h5py_root[qa_item_path]
                 ~~~~~~~~~~~~~~^^^^^^^^^^^^^^
  File "h5py/_objects.pyx", line 54, in h5py._objects.with_phil.wrapper
  File "h5py/_objects.pyx", line 55, in h5py._objects.with_phil.wrapper
  File "/mnt/aurora-r0/vbrancat/codes/miniconda3/envs/compass/lib/python3.11/site-packages/h5py/_hl/group.py", line 357, in __getitem__
    oid = h5o.open(self.id, self._e(name), lapl=self._lapl)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "h5py/_objects.pyx", line 54, in h5py._objects.with_phil.wrapper
  File "h5py/_objects.pyx", line 55, in h5py._objects.with_phil.wrapper
  File "h5py/h5o.pyx", line 190, in h5py.h5o.open
KeyError: 'Unable to open object (component not found)'

```

**Comment by vbrancat** [comment]
@seongsujeong please review and test this PR as well

**Review by seongsujeong — APPROVED** [review]
LGTM. I don't see any issues from the test results on my end.

**Review by vbrancat — APPROVED** [review]
Structure looks good to me. Let's make sure the actual raster layers look good as well when plotted

**Comment by LiangJYu** [comment]
3 of the 4 default static layers look good.
image
Shadow layover can't correctly assign the correct invalid value. This is a know [ISCE3 issue](https://github-fn.jpl.nasa.gov/isce-3/isce/issues/1301) and can't be corrected in time for cal val.
image

**Pull request merged**